.gitignore

```diff
@@ -25,3 +25,4 @@ build/
 
```

Comment:
everything looks good for the most part.  Consider adding your package-lock.json and your csv files in your git ignore

Reply:
Thanks. I added these example*.csv files so user can know the shape of the data.